python_docs_theme/__init__.py

```diff
@@ -12,9 +14,22 @@
 THEME_PATH = Path(__file__).resolve().parent
 
 
+def setup_translations(app):
+    translation = gettext.translation(
```

Comment:
Should we use [`sphinx.locale.get_translation()`](https://www.sphinx-doc.org/en/master/extdev/i18n.html#sphinx.locale.get_translation) here? 

https://www.sphinx-doc.org/en/master/extdev/i18n.html#extension-internationalization-i18n-and-localization-l10n-using-i18n-api